packages/edit-post/src/components/keyboard-shortcut-help-modal/test/__snapshots__/index.js.snap

```diff
@@ -1,106 +1,381 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`KeyboardShortcutHelpModal should match snapshot when the modal is active 1`] = `
-<ForwardRef(Modal)
-  className="edit-post-keyboard-shortcut-help-modal"
-  closeLabel="Close"
```

Comment:
Just noting that in the old snapshot, `closeLabel` had a value of `Close`, but in the new snapshot the label of the close buttons seems to be `Close dialog`. Perhaps it's a bug that we should look at?

Reply:
Oh, that's a great find! 💯 

Seems like it was caused by the wrong close button label prop used in the original component. 🐛 

I've fixed that in 68aaa0e7971ffd24f6ef4378ebabdc2aaa2bafea.